DS2S META/Utils/DS2Hook/DS2SHook.cs

```diff
@@ -1659,7 +1669,48 @@ public void SetDisableSkirtDamage(bool disableSkirtDamage)
                 UninstallDisableSkirtDamage();
         }
 
-        
+        public void SetDisablePartyWalkTimer(bool disablePartyWalkTimer)
```

Comment:
There might be a better spot to write this function instead of directly in the DS2SHook but i am unsure where